### PR TITLE
[zh-cn]sync reload-kubelet-server-certificate-file

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+---
+
+<!--
+Enable the kubelet TLS server to update its certificate if the specified certificate file are changed.
+
+This feature is useful when specifying `tlsCertFile` and `tlsPrivateKeyFile` in kubelet configuration.
+The feature gate has no effect for other cases such as using TLS boostrap.
+-->
+启用 kubelet TLS 服务器在指定的证书文件发生变化时更新其证书
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md:21]:
```suggestion
允许 kubelet TLS 服务器在指定的证书文件发生变化时更新其证书。
```

**Review by windsonsea — COMMENTED** [review]
nit.
/approve

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+此特性在 kubelet 配置中指定 `tlsCertFile` 和 `tlsPrivateKeyFile` 时非常有用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md:23]:
```suggestion
此特性在 kubelet 配置中指定了 `tlsCertFile` 和 `tlsPrivateKeyFile` 时非常有用。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+对于其他情况，例如使用 TLS 自动启动（bootstrap），此特性门控无效。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/reload-kubelet-server-certificate-file.md:24]:
```suggestion
对于其他情况，例如使用 TLS 引导启动时，此特性门控无效。
```

**Pull request merged**